performance-storage-service/pss_project/api/models/rest/metadata/GithubMetadata.py

```diff
@@ -1,5 +1,5 @@
 class GithubMetadata(object):
-    def __init__(self, commit_id, branch):
-        self.commit_id = commit_id 
-        self.branch = branch
+    def __init__(self, git_commit_id, git_branch):
+        self.git_commit_id = git_commit_id 
+        self.git_branch = git_branch
```

Comment:
since this is part of the GitHubMetadata class we probably don't need the `git_` on the attributes

Reply:
> since this is part of the GitHubMetadata class we probably don't need the `git_` on the attributes

I changed the name so that the name here matches the column name and it won't cause possible confusion.